platforms/documentation/docs/src/docs/userguide/authoring-builds/build_environment.adoc

```diff
@@ -131,54 +129,82 @@ spring-boot.version = 2.2.1.RELEASE
 [[sec:project_properties]]
 == Project properties
 
-Project properties are specific to your Gradle project, they are variables or configuration blocks defined in your build script or in the `project` object.
+Project properties are specific to your Gradle project, they are variables or configuration blocks defined in your build script or in the `Project` object.
```

Comment:
This is a little strange to me because you don't define project properties in build scripts. They are defined using one of the 4 options shown below (command line, properties file, environment variables). Maybe "defined" is not the best word to use here, maybe "referenced"? 

Also, what does it mean "or configuration blocks" and "or in the `Project` object"?

Reply:
It is a poor choice of words, defined should be created. Rewording this.